data.sql

```diff
@@ -156,7 +156,6 @@ Boarmon visited Maisy Smith on Feb 27th, 2020.
 Boarmon visited Maisy Smith on Aug 3rd, 2020.
 Blossom visited Stephanie Mendez on May 24th, 2020.
 Blossom visited William Tatcher on Jan 11th, 2021.
-*/
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍